google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -665,11 +665,23 @@ def create_file file, path = nil, acl: nil, cache_control: nil,
         #                                  issuer: "service-account@gcloud.com",
         #                                  signing_key: key
         #
+        # @example Using the headers option:
+        #   require "google/cloud/storage"
+        #
+        #   storage = Google::Cloud.storage
+        #
+        #   bucket = storage.bucket "my-todo-app"
+        #   shared_url = bucket.signed_url "avatars/heidi/400x400.png",
+        #                                  headers: {
+        #                                    "x-goog-acl" => "private",
+        #                                    "x-goog-meta-foo" => "bar,baz"
```

Comment:
@blowmage You changed the headers in the acceptance test to be capitalized (`X-Goog-ACL`), do you want to do it as well in these docs examples?